### PR TITLE
[front] feat: Add language selector in footer

### PR DESCRIPTION
**Pull request by amatissart**
To be merged after #450 

image

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "react-scripts": "^5.0.0",
     "redux": "^4.1.2",
     "redux-persist": "^6.0.0",
-    "typescript": "~4.1.5",
+    "typescript": "~4.5.4",
```
Comment (amatissart) [inline frontend/package.json:30]: Upgrade required in order to type correctly `Int.DisplayNames` (used to get language names)

File: frontend/src/features/settings/account/ExportAllDataForm.tsx [hunk 2]
```diff
@@ -44,7 +44,7 @@ const ExportAllDataForm = () => {
     const response = await usersMeExportsAllRetrieveBlob();
 
     if (response.ok === true) {
-      const zipFileBlob = response.blob();
+      const zipFileBlob = await response.blob();
```
Comment (amatissart) [inline frontend/src/features/settings/account/ExportAllDataForm.tsx:47]: Fixes a Typescript error (not sure why it was not triggered by the previous version).

File: frontend/src/components/LanguageSelector.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    <Box color="neutral.main">
+      <MenuButton
+        startIcon={<Language />}
+        sx={{ width: '100%', px: 3 }}
+        menuContent={SUPPORTED_LANGUAGES.map((lang) => (
+          <MenuItem
+            key={lang}
+            sx={{ minWidth: '200px' }}
+            onClick={() => i18n.changeLanguage(lang)}
+          >
```
Comment (lfaucon) [inline frontend/src/components/LanguageSelector.tsx:29]: Could this show which language is selected?

**Review by lfaucon — CHANGES_REQUESTED** [review]

**Review by lfaucon — APPROVED** [review]
💯

**Pull request merged**